targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52832/device/TOOLCHAIN_ARM_STD/nRF52832.sct

```diff
@@ -9,17 +9,6 @@
   #define MBED_APP_SIZE 0x80000
 #endif
 
-/* If softdevice is present, set aside space for it */
```

Comment:
What has changed that we don't need to block space for softdevice anymore?

Reply:
The new memory stats collection (specifically https://github.com/ARMmbed/mbed-os/pull/8607) broke Nordic SD builds as it defines MBED_RAM_START so the scatterfile needed to change.  We now define MBED_RAM_START and MBED_RAM_SIZE in the library json file when building with the SD:

https://github.com/TacoGrandeTX/mbed-os/blob/chile/targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_SDK_15_0/TARGET_SOFTDEVICE_S132_FULL/mbed_lib.json#L37-L39

This works out good because if one changes the BLE config the RAM requirements might also change and then they only need to touch one file.

Reply:
We have only done on NRF52_DK and NRF52840_DK for now, I think that means we should add NRF51_DK to the verification, thank you for comments on this.

Reply:
@desmond-blue SDK15 doesn't include support for the NRF51 so there is no need to verify on the NRF51_DK.